Classes/git/PBGitGraphLine.h

```diff
@@ -8,8 +8,8 @@
 
 struct PBGitGraphLine
 {
-	int upper      : 1;
+	int upper       : 1;
 	int from       : 8;
 	int to         : 8;
-	int colorIndex : 8;
+	int colorIndex  : 8;
```

Comment:
Nitpick: `long` was reverted but the indentation change on the other lines remains.